doc/api/keys/keys.yaml

```diff
@@ -0,0 +1,176 @@
+              schema:
+                $ref: '#/components/schemas/ErrorInformation'
+                
+  /evidence/{ChallengeId}:
+    post:
+      description: >
+        Provide evidence in response to a given challenge.
+      parameters:
+        - $ref: '#/components/parameters/ChallengeId'
+      requestBody:
+        required: true
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/EvidenceBytes'
+
+      responses:
+        200:
+          description: Wrapped key data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WrappedKeyData'
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInformation'
+
+components:
+  parameters:
+    KeyId:
+      name: KeyId
+      in: path
+      required: true
+      schema:
+        type: string
+    
+    ChallengeId:
+      name: ChallengeId
+      in: path
+      required: true
+      schema:
+        type: string
+
+  schemas:
+    BackgroundCheckKeyRequest:
+      required:
+        - pubkey
+      properties:
+        pubkey:
+          $ref: '#/components/schemas/PublicWrappingKey'
+          description: Public key-wrapping key
+
+    AttestationChallenge:
+      required:
+        - challenge
+        - accept
+      properties:
+        challenge:
+          type: string
+          format: byte
+          description: >
+            Base64-encoded challenge value (nonce) for the attester. The attester
+            must include this nonce in the signed evidence.
+        accept:
+          type: array
+          items:
+            $ref: '#/components/schemas/EvidenceContentType'
+            description: >
+              Acceptable MIME types for attestation Evidence submission. The attester
+              must provide evidence of one of these types.
+
+    EvidenceBytes:
+      type: string
+      example: 'eyJhbGciO...RfrKmTWk'
+      description: >
+        Base64 encoded Evidence
+
+    EvidenceContentType:
+      type: string
+      example: 'application/psa-attestation-token'
+
+    WrappedKeyData:
+      required:
+        - data
+      properties:
+        data:
+          type: string
+          format: byte
+          description: >
+            Key data, wrapped using the public key that was provided in the initial key request.
+
+    PublicWrappingKey:
+      required:
+        - kty
+        - alg
+        - n
+        - e
+      properties:
+        kty:
+          type: string
+          description: Key Type
+        alg:
+          type: string
+          description: Key Algorithm
+        n:
+          type: string
+          description: Key modulus
+        e:
+          type: string
+          description: Key exponent
+      description: >-
+        A JSON Web Key (https://www.rfc-editor.org/rfc/rfc7517) formatted RSA Public Key.
```

Comment:
Makes total sense.  ECIES is not (yet) well supported it seems, at least go is behind.